src/fullscreen.ts

```diff
@@ -50,7 +50,7 @@ export class FullscreenButtonView extends DOMWidgetView {
     this.el.classList.add('jupyter-widgets');
     this.el.classList.add('jupyter-button');
     this.el.classList.add('widget-button');
-    this.delegate('click', this.el, this._handle_click);
+    this.delegate('click', this.el.value, this._handle_click);
```

Comment:
got a compilation error without this complaining about assigning a string output to a `HTMLButtonElement`, based on some google/SO searching, seemed like this would work. and it does now compile. though just realized I didn't test the button, will go do that now ...

Reply:
hmm. ok, ya, i don't even see the fullscreen button any more, so i broke something here.

Reply:
I don't think it totally works yet.  So let's not worry about this for this PR and fix it at a later time.  If I can figure it out I'll update your PR.